elastic/kibana
# [Obs AI Assistant] Fix AI Assistant Settings when plugin disabled

## Summary

Fixes https://github.com/elastic/kibana/issues/187977

AI Assistant settings page doesn't load when the plugin is disabled.  

### Changes

- fixes not checking for object before accessing property
- simplifies message so it makes sense for all possible states (read only, disabled in space, plugin disabled)

### Testing steps
- create a Viewer user
- go to Stack Management -> Advanced Settings -> AI Assistant
- page should load with message that the feature is disabled

<img width="804" alt="Screenshot 2024-07-11 at 4 57 59 PM" src="https://github.com/user-attachments/assets/98cf85a2-913f-43fd-a5f0-df4494faaf1f">
